Code.gs

```diff
@@ -12,12 +12,15 @@
 // var targetCalendarName = "" // The name of the Google Calendar you want to add events to
```

Comment:
If we're going to deprecate `targetCalendarName` and `sourceCalendarURL`, let's just remove those options